### PR TITLE
adding font family and change dir files

### PR DESCRIPTION
**Pull request by KatiaAsh**
relates #8

### CODE REVIEW

**Review by tyeung2017 — CHANGES_REQUESTED** [review]
please merge with master before push.

right now the trait page you have in your version is still the old version

And the result page is not linked to arabic trait page

Also @wyrd can you help have a look of the css in Arabic page?

**Comment by KatiaAsh** [comment]
@tyeung2017 
I merge with master

Trait page is an Arabic file I didn't add the new version
(ill make new PR for Ar trait page :+1: )

And I didn't change anything in CSS only added font family
(and I use "atom beautify")

Can you show me where should i link result to trait page?

**Review by tyeung2017 — APPROVED** [review]
@Wyrd00 maybe we can merge this first and fix the css and other stuff later

**Review by Wyrd00 — APPROVED** [review]

File: game.css [hunk 1]
```diff
@@ -1,16 +1,21 @@
+* {
+  font-family: 'Lateef', cursive;
+  font-size: 22px;
```
Comment (Karyum) [inline game.css]: use rem for a responsive design

File: game.css [hunk 2]
```diff
@@ -25,57 +30,57 @@ button{
...
-#div-balloon-counter{
+#div-balloon-counter {
   text-align: center;
-  flex-grow:1;
+  flex-grow: 1;
   font-size: 1.5rem;
 }
 
-#canvas{
+#canvas {
```
Comment (Karyum) [inline game.css]: not sure you need this anymore since it's commented out

File: game.css [hunk 3]
```diff
@@ -25,57 +30,57 @@ button{
...
 }
 
-#collect{
+#collect {
   color: white;
   background-color: #42f4a7;
 }
 
-#bag-of-money{
+#bag-of-money {
```
Comment (Karyum) [inline game.css:77]: and here

File: instruction.css [hunk 4]
```diff
@@ -1,3 +1,8 @@
+* {
+  font-family: 'Lateef', cursive;
+  font-size: 22px;
```
Comment (Karyum) [inline instruction.css]: use rem

File: navbar.css [hunk 5]
```diff
@@ -1,32 +1,37 @@
-nav{
+* {
+  font-family: 'Lateef', cursive;
+  font-size: 22px;
```
Comment (Karyum) [inline navbar.css]: rem

File: result.css [hunk 6]
```diff
@@ -1,3 +1,8 @@
+* {
+  font-family: 'Lateef', cursive;
+  font-size: 22px;
```
Comment (Karyum) [inline result.css]: and rem :maple_leaf:

File: result.css [hunk 7]
```diff
@@ -20,11 +25,11 @@ body {
 }
 
 img {
-/*  margin-left: auto;
+  /*  margin-left: auto;
```
Comment (Karyum) [inline result.css]: commented out code

File: traits.css [hunk 8]
```diff
@@ -1,64 +1,89 @@
-body{
+* {
+  font-family: 'Lateef', cursive;
+  font-size: 22px;
```
Comment (Karyum) [inline traits.css]: guess what im going to say here

**Review by Karyum — DISMISSED** [review]
sorry about that

File: game.css (hunk 1 above)
Comment (KatiaAsh) [reply game.css]: Mario :angry: 
I normally use but in this section, i can't 
the font will be so big and i need to write something like that "0.1234rem" :thinking:

File: game.css (hunk 2 above)
Comment (KatiaAsh) [reply game.css]: @tyeung2017

File: game.css (hunk 3 above)
Comment (KatiaAsh) [reply game.css]: @tyeung2017

File: instruction.css (hunk 4 above)
Comment (KatiaAsh) [reply instruction.css]: Can't :person_frowning:

File: navbar.css (hunk 5 above)
Comment (KatiaAsh) [reply navbar.css]: Can't :person_frowning:

File: result.css (hunk 6 above)
Comment (KatiaAsh) [reply result.css]: Can't :person_frowning:

File: result.css (hunk 7 above)
Comment (KatiaAsh) [reply result.css]: @tyeung2017 @Wyrd00

File: traits.css (hunk 8 above)
Comment (KatiaAsh) [reply traits.css]: guess what I'm going to reply here :joy:

**Review by Karyum — COMMENTED** [review]
I dismissed my last review cause why do you even need the font size thing, also please remove the commented out code

File: game.css (hunk 3 above)
Comment (KatiaAsh) [reply game.css:77]: This one (for later use)

**Comment by tyeung2017** [comment]
@KatiaAsh ok that works as well

**Comment by KatiaAsh** [comment]
sure :+1:

**Review by Karyum — APPROVED** [review]

**Pull request merged**